addon/components/paper/number-field.js

```diff
@@ -0,0 +1,37 @@
+import { get } from '@ember/object';
+import { isBlank }  from '@ember/utils';
+
+import PaperTextField from
+  'ember-railio-form-components/components/paper/text-field';
+import { toNumber, formatNumber } from 'ember-railio-formatting';
+
+function increaseNumber(value, add) {
+  value = toNumber(value);
+  return isBlank(value) ? 0 : value + add;
+}
+
+export default PaperTextField.extend({
+  layoutName: 'components/paper-text-field',
+
+  decimals: 2,
+  lazy:     true,
+
+  keyDown(e) {
+    let value = get(this, '_value') || 0;
+
+    let keyValue = e.keyCode === 38 ? 1  : 0 ||
```

Comment:
`KeyboardEvent.keyCode` is deprecated, use `KeyboardEvent.key` instead. This will also improve readablity, as the values for up and down arrow keys are `ArrowUp` and `ArrowDown`